conf/portal.application.conf

```diff
@@ -187,14 +187,14 @@ hostProvider {
 # Alerts
 # ~~~~~
 alertRepository {
-  type = com.arpnetworking.metrics.portal.alerts.impl.NoAlertRepository
+  type = com.arpnetworking.metrics.portal.alerts.impl.DatabaseAlertRepository
```

Comment:
Should we not encourage people to use them now?  Should I also turn on those feature flags?